src/PerfectGym.AutomergeBot/Features/CodeReviewInspector/CodeReviewInspector.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Octokit;
+using PerfectGym.AutomergeBot.Notifications.SlackClient;
+using PerfectGym.AutomergeBot.Notifications.UserNotifications;
+using PerfectGym.AutomergeBot.RepositoryConnection;
+
+namespace PerfectGym.AutomergeBot.Features.CodeReviewInspector
+{
+    public class CodeReviewInspector
+    {
+        private readonly IRepositoryConnectionProvider _repositoryConnectionProvider;
+        private readonly IUserNotifier _userNotifier;
+        private readonly ILogger<CodeReviewInspector> _logger;
+        private readonly AutomergeBotConfiguration _cfg;
+        TimeScheduler  _timeScheduler= new TimeScheduler();
+
+        public CodeReviewInspector(
+            ILogger<CodeReviewInspector> logger,
+            IOptionsMonitor<AutomergeBotConfiguration> cfg,
+            IRepositoryConnectionProvider repositoryConnectionProvider,
+            IUserNotifier userNotifier
+            )
+        {
+            _logger = logger;
+            _repositoryConnectionProvider = repositoryConnectionProvider;
+            _userNotifier = userNotifier;
+            _cfg = cfg.CurrentValue;
+        }
+
+        public void StartWorker()
+        {
+            if (_cfg.CodeReviewInspectorConfiguration != null && _cfg.CodeReviewInspectorConfiguration.IsEnable)
+            {
+                foreach (var time in _cfg.CodeReviewInspectorConfiguration.SendNotificationsToReviewersAtTimes)
+                {
+                    if (TimeSpan.TryParse(time, out TimeSpan timeAsTimeSpan))
+                    {
+                        _timeScheduler.ScheduledActions.Add(new TimeScheduler.ScheduleAction(timeAsTimeSpan, CheckOpenPullRequestsAndNotifyOwnersAndReviewers));
+                    }
+                    else
+                    {
+                        _logger.LogWarning("Could not parse time ({valeu}) form SendNotificationsToReviewersAtTimes. Expected format HH:MM", time);

```

Comment:
Coś tutaj zupełnie nie działa. To jest error.
Jeśli uważamy że błędy komponentu "CodeReviewInspector" są mało ważne to należy to przemapować na innym poziomie. (NLog config chyba umożliwia coś takiego)

```suggestion
                        _logger.LogError("Could not parse time ({value}) of configuration property '"+ nameof(CodeReviewInspectorConfiguration.SendNotificationsToReviewersAtTimes) +"'. Expected format HH:MM", time);
```